src/terminal_pane/terminal_pane.rs

```diff
@@ -733,12 +735,9 @@ impl vte::Perform for TerminalPane {
                 _ => false,
             };
             if first_intermediate_is_questionmark {
-                match params.get(0) {
-                    Some(25) => {
-                        self.scroll.show_cursor();
-                        self.mark_for_rerender();
-                    }
-                    _ => {}
+                if let Some(&25) = params.get(0) {
+                    self.scroll.show_cursor();
+                    self.mark_for_rerender();
                 };
             }
         } else if c == 'r' {
```

Comment:
@5c077m4n - I can't comment on the lines below directly for some reason, but can we disable this clippy check? The one with the errors about the identical ifs. I would really like to leave it like this for now. The comments are super important and this block is generally very much in-flux as we keep reverse-engineering the VT100 protocol.